### PR TITLE
Chinese blog of comparing MP and Spring

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Fixes: #2638

### CODE REVIEW

**Comment by Emily-Jiang** [comment]
I tried to add a few images as well. However, the images was not displayed in the blog. Please take a look to see whether I have done wrong.

File: posts/2022-10-16-microprofile-spring-comparison-in-chinese.adoc [hunk 1]
```diff
@@ -0,0 +1,1057 @@
...
+创建微服务。有关如何使用命令行工具，请参阅link:https://microprofile.io/2019/07/08/command-line-interface-for-microprofile-starter-is-available-now/[Karm
+的博客]我们提供 VS Code 和 Intellij扩展插件，以允许您直接从您的 IDE
+创建微服务。我们计划为其他 IDE 创建扩展，例如Eclipse IDE 等。敬请期待！
+
+=== Spring Starter
+
+Spring 有一个起始页面 ( https://start.spring.io/ ) 可帮助您创建 Spring
+Boot 应用程序。
+
+image::/img/blog/sspring_starter.png[SpringBoot starter, width=70%,align="center"]
```
Comment (dmuelle) [inline posts/2022-10-16-microprofile-spring-comparison-in-chinese.adoc]:
```suggestion
image::/img/blog/spring_starter.png[SpringBoot starter, width=70%,align="center"]
```

File: posts/2022-10-16-microprofile-spring-comparison-in-chinese.adoc [hunk 2]
```diff
@@ -0,0 +1,1057 @@
...
+MicroProfile和Spring 都有一个起始页。
+
+=== MicroProfile 入门
+
+MicroProfile starter ( https://start.microprofile.io/ )
+为您提供了一种使用 MicroProfile
+创建微服务的好方法，您可以选择自己喜欢的运行容器，例如link:https://openliberty.io/[Open
+Liberty]、link:https://thorntail.io/[Thorntail]、link:https://www.payara.fish/[Payara]、link:https://tomee.apache.org/[TomEE]、link:https://ee.kumuluz.com/[KumuluzEE]、link:https://helidon.io/[Helidon]等.
+
+image::img/blog/MP_starter.png[MicroProfile Starter, width=70%,align="center"]
```
Comment (dmuelle) [inline posts/2022-10-16-microprofile-spring-comparison-in-chinese.adoc]:
```suggestion
image::/img/blog/MP_starter.png[MicroProfile Starter, width=70%,align="center"]
```

**Comment by dmuelle** [comment]
Hi Emily- I found a couple typos in the image URLs. I fixed them and the images are showing in the PR preview now.

**Comment by Emily-Jiang** [comment]
Awessome, thanks @dmuelle !

**Comment by kinueng** [comment]
There are a few things we need to do on the `openliberty.io` website code to support this blog post. I will push this through to production when `openliberty.io` is ready for this PR.

**Review by kinueng — CHANGES_REQUESTED** [review]
@kinueng and team need to deliver code before this PR can be merged.

This PR is blocked by https://github.com/OpenLiberty/openliberty.io/issues/2931

**Pull request merged**